setup.py

```diff
@@ -53,6 +53,10 @@
             "sphinx-thebe",
         ],
         "testing": [
+            "myst_nb~=0.10.1",
+            "sphinx_copybutton",
```

Comment:
I _think_ we can remove both the copybutton/togglebutton from the tests (and the `conf.py` in the site that the tests build)